rom/dev/tests/rom_integration_tests/test_idevid_derivation.rs

```diff
@@ -54,3 +67,56 @@ fn test_idev_subj_key_id_algo() {
             .unwrap();
     }
 }
+
+fn fuses_with_random_uds() -> Fuses {
+    const UDS_LEN: usize = core::mem::size_of::<u32>() * 12;
+    let mut uds_bytes = [0; UDS_LEN];
+    rand_bytes(&mut uds_bytes).unwrap();
+    let mut uds_seed = [0u32; 12];
+
+    for (word, bytes) in uds_seed.iter_mut().zip(uds_bytes.chunks_exact(4)) {
+        *word = u32::from_be_bytes(bytes.try_into().unwrap());
+    }
+    Fuses {
+        uds_seed,
+        ..Default::default()
+    }
+}
+
+#[test]
+fn test_generate_csr_stress() {
+    #[cfg(feature = "slow_tests")]
+    const NUM_TESTS: usize = 1000;
+    #[cfg(not(feature = "slow_tests"))]
+    const NUM_TESTS: usize = 1;
```

Comment:
nit: Should be able to do something like

```rust
let num_tests: usize = if cfg!(feature = "slow_tests") {
    1000
} else {
    1
}
```

Reply:
That's a lot better